### PR TITLE
build: Support 1ES pipelines

### PR DESCRIPTION
**Pull request by bwateratmsft**

### CODE REVIEW

File: azure-pipelines/templates/1espackage.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+steps:
+  - task: Npm@1
+    displayName: "\U0001F449 Package"
```
Comment (bwateratmsft) [inline azure-pipelines/templates/1espackage.yml:3]: This is the 👉 emoji. It makes it much easier to visually separate what is from our template vs. what is from 1ES templates.

File: azure-pipelines/templates/1espackage.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+steps:
+  - task: Npm@1
+    displayName: "\U0001F449 Package"
+    inputs:
+      command: custom
+      customCommand: run package
+    condition: succeeded()
+
+  - task: CopyFiles@2
```
Comment (bwateratmsft) [inline azure-pipelines/templates/1espackage.yml:9]: 1ES has a publish task built in, you just have to copy the files to the publish folder

File: azure-pipelines/templates/package.yml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 steps:
```
Comment (MRayermannMSFT) [inline azure-pipelines/templates/package.yml:1]: So is this file being used anywhere anymore?

Comment (bwateratmsft) [reply azure-pipelines/templates/package.yml:1]: It's still being used by the old pipelines which I'd like to keep alive and working while we work on this.

**Review by MRayermannMSFT — APPROVED** [review]
LGTM. Probably get at least one other azcode folk to review though. 🙈

**Comment by nturinski** [comment]
So for clarification, does this mean that we'll stop publishing our @microsoft packages to npm and have to use the 1ES registry or whatever the equivalent is?

**Comment by bwateratmsft** [comment]
> So for clarification, does this mean that we'll stop publishing our @microsoft packages to npm and have to use the 1ES registry or whatever the equivalent is?

We still plan to publish to NPM, but we have to do release through ESRP instead of via our own accounts / our own bot account.

**Pull request merged**